blackbox/docs/config/logging.rst

```diff
@@ -1,34 +1,57 @@
-.. highlight:: sh
-
 .. _conf-logging:
 
 =======
 Logging
 =======
 
+CrateDB supports two kinds of logging:
+
+- Application logging with `Log4j`_
+- Java Virtual Machine (JVM) garbage collection logging
+
+We use "application" here to distinguish between CrateDB running as a Java
+application and the JVM itself, which runs CrateDB.
+
+Because garbage collection logging is a native feature of the JVM it behaves
+differently and is configured differently.
+
 .. rubric:: Table of Contents
 
 .. contents::
    :local:
 
+.. _conf-logging-app:
+
 Application Logging
 ===================
 
-CrateDB comes, out of the box, with Log4j_ 1.2.x. It tries to simplify log4j
-configuration by using YAML to configure it. The logging configuration file is
-located at ``$CRATE_HOME/config/log4j2.properties``.
+.. _conf-logging-log4j:
+
+Log4j
+-----
+
+CrateDB uses `Log4j`_ 1.2.x.
```

Comment:
Actually, as far as I can see, both log4j and log4j2 are supported but the default file is in log4j2 format.